docs/_templates/search.html

```diff
@@ -15,173 +15,13 @@
-
-                if (totalPages - currentPage < nbPagesLinkToDisplay) {
-                    start = totalPages - nbPagesLinkToDisplay
-                }
-
-                if (totalPages - start > nbPagesLinkToDisplay) {
-                    end = currentPage + nbPagesLinkToDisplay
-                }
-            }
-
-            return [start, end]
-        }
-
-        function addHeadingForQuery(query, parentElement) {
-            const searchHeading = document.createElement('h1')
-            searchHeading.textContent = `Search results for “${query}”`
-            parentElement.appendChild(searchHeading)
-        }
-
-        function addResultsList(hits, query, parentElement) {
-            const searchResultsList = document.createElement('ul')
-            searchResultsList.className = "search"
-            for (hit of hits) {
-                const hitElement = createHitElement(hit, query)
-                searchResultsList.appendChild(hitElement)
-            }
-            parentElement.appendChild(searchResultsList)
-        }
-
-        function createHitElement(hitData, query) {
-            const pageURL = new URL(hitData.url)
-            pageURL.hash = '';
-            pageURL.searchParams.set('highlight', query);
-            const anchorURL = new URL(hitData.url);
-            anchorURL.searchParams.set('highlight', query);
-            const result = hitData._highlightResult
-
-            const hitListElement = document.createElement('li')
-
-            const hierarchies = Object.values(result.hierarchy);
-            const firstHierarchyLevel = hierarchies[0];
-            const lastHierarchyLevel = hierarchies[hierarchies.length - 1];
-
-            const pageLink = document.createElement('a')
-            hitListElement.appendChild(pageLink)
-            pageLink.innerHTML = firstHierarchyLevel.value
-            pageLink.href = pageURL
-
-            const contextElement = document.createElement('div')
-            hitListElement.appendChild(contextElement)
-            contextElement.className = 'context'
-
-            if (lastHierarchyLevel && lastHierarchyLevel !== firstHierarchyLevel ) {
-                const contextLinkContainer = document.createElement('div')
-                contextElement.appendChild(contextLinkContainer)
-                const contextLink = document.createElement('a')
-                contextLinkContainer.appendChild(contextLink)
-                contextLink.innerHTML = lastHierarchyLevel.value
-                contextLink.href = anchorURL
-            }
-
-            if (result.content) {
-                const contentElement = document.createElement('div')
-                contentElement.innerHTML = result.content.value
-                contextElement.appendChild(contentElement)
-            }
-
-            return hitListElement
-        }
-
-        window.addEventListener('DOMContentLoaded', () => runSearchPageSearch(0));
-    </script>
-{% endblock %}
+{% endblock %}
```

Comment:
note - it appears your editor has removed a line break at the end of the file, please can you re-add it.

Reply:
Couldn't find the line break. Apologies in case I am missing something.

Reply:
Ok maybe the line break was there but has been removed. Not a problem for this PR right now.